frontend/packages/knative-plugin/src/components/overview/FilterTable.scss

```
@@ -0,0 +1,6 @@
+.kn-filter-table-row-bordered {
+  border: var(--pf-global--BorderWidth--sm) solid var(--pf-global--palette--black-150);
+}
+.kn-filter-table-padding-left {
+  padding-left: var(--pf-global--spacer--3xl) !important;
+}
```

`!important` not required.

```suggestion
.kn-filter-table {
  &--row-bordered {
    border: var(--pf-global--BorderWidth--sm) solid var(--pf-global--palette--black-150);
  }
  &--padding-left {
    padding-left: var(--pf-global--spacer--3xl);
  }
}
```